README.md

```diff
@@ -1,6 +1,63 @@
 # Temporal Ruby SDK
 
-The Ruby SDK is under active development/refresh.
+⚠️ UNDER ACTIVE DEVELOPMENT
 
 The last tag before this refresh is [v0.1.1](https://github.com/temporalio/sdk-ruby/tree/v0.1.1). Please reference that
-tag for the previous code.
\ No newline at end of file
+tag for the previous code.
+
+**TODO: Usage documentation**
+
+## Development
+
+### Build
+
+Prerequisites:
+
+* [Ruby](https://www.ruby-lang.org/) (i.e. `ruby` and `bundle` on the `PATH`)
+* [Rust](https://www.rust-lang.org/) (i.e. `cargo` on the `PATH`)
+* [Protobuf Compiler](https://protobuf.dev/) (i.e. `protoc` on the `PATH`)
+* This repository, cloned recursively
+* Change to the `temporalio/` directory
```

Comment:
We chose to move this to a subdir because we expect other gems (e.g. `temporalio-opentelemetry`) and we don't want too many top-level files that move the README contents below the fold when arriving at the repo.